tools/shared.py

```diff
@@ -518,7 +518,7 @@ def get_emscripten_version(path):
 # For the Emscripten-specific WASM metadata section, follows semver, changes
 # whenever metadata section changes structure
 # NB: major version 0 implies no compatibility
-(EMSCRIPTEN_METADATA_MAJOR, EMSCRIPTEN_METADATA_MINOR) = (0, 1)
+(EMSCRIPTEN_METADATA_MAJOR, EMSCRIPTEN_METADATA_MINOR) = (0, 2)
```

Comment:
Shouldn't this be a major version change? From the way the comments read, minor version is for appending fields. This makes sense, since code looking for `mem_size` will now instead see `Settings.WASM_BACKEND`.

Reply:
@quantum5 I think as the comment implies, 0 for the major version implies any change can break stuff. And I don't think we are "stable" enough in any sense to write "1" for the major versions, which suggests stability I believe.